sympy/logic/boolalg.py

```diff
@@ -914,6 +914,16 @@ def to_nnf(self, simplify=True):
         a, b, c = self.args
         return And._to_nnf(Or(~a, b), Or(a, c), simplify=simplify)
 
+    def _eval_derivative(self, x):
+        return self.func(self.args[0], *[a.diff(x) for a in self.args[1:]])
+
+    # the diff method below is copied from Expr class
```

Comment:
Are you suggesting that ITE should inherit from Expr? I just haven't thought about all implications of that.